tests/mockers.py

```diff
@@ -484,3 +484,43 @@ def mock_derive_current_user_login(domain: str) -> str:
 def mock_input(msg: str) -> str:
     print(msg)
     return '1'
+
+
+def mock_is_file_true(file: Any) -> bool:
+    return True
+
+
+def mock_is_file_false(file: Any) -> bool:
+    return False
+
+
+def mock_is_file_not_github_token(file: Any):
+    if '.github-token' not in file:
+        return True
+
+
+def mock_os_environ_get_none(self, key: str, default=None) -> Optional[str]:
+    if key == GitHubToken.GITHUB_TOKEN_ENV_VAR:
+        return None
+    try:
+        return self[key]
+    except KeyError:
+        return default
+
+
+def mock_os_environ_get_github_token(key: str, default=None) -> Optional[str]:
+    return 'github_token_from_env_var'
+
+
+def mock_iterate_read_data(read_data):
+    data_as_list = read_data.split('\n')
+    for line in data_as_list:
+        yield line
+
+
+def mock_shutil_which_gh(cmd):
+    return 'path_to_gh_executable'
+
+
+def mock_subprocess_run(*args, stdout, stderr):
```

Comment:
Same remark, pls don't hardcode any non-obvious assumption into supposedly generic mockers

Reply:
Or alternatively: maybe instead of keeping these mockers in common module like here, keep them inside the file where they're used (test_github.py)

Reply:
@PawelLipski right, i forgot mocker.py is only for generic mockers, I will move the files into test_github.py